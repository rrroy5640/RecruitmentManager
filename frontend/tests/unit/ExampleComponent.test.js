import React from 'react';
import { render } from '@testing-library/react';

describe('ExampleComponent', () => {
  it('should render correctly', () => {
    //render something
    const { container } = render(<h1>Hello, World!</h1>);
    expect(container).toMatchSnapshot();
  });
});
