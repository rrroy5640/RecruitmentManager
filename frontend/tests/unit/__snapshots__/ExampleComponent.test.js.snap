// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExampleComponent should render correctly 1`] = `
<div>
  <h1>
    Hello, World!
  </h1>
</div>
`;
